target/linux/octeon/base-files/lib/preinit/79_move_config

```diff
@@ -10,6 +10,22 @@ move_config() {
 			[ -f "/mnt/$BACKUP_FILE" ] && mv -f "/mnt/$BACKUP_FILE" /
 			umount /mnt
 			;;
+		itusrouter)
```

Comment:
Couldn't you have just one case here for `itusrouter|itusbridge|itusgateway` ?

Reply:
Yes, i was splitting them out for testing, but i don't know why I couldn't do a "itus"* there
Will correct

Reply:
(from IRC discussion) ... because you actually want them to have different filenames like `/mnt/${BACKUP_FILE}-${board}` to prevent an upgrade intended for one partition being ingested by another partition if the mode switch is flipped at the wrong time.

Reply:
No itus* please.
Just
```
itusbridge|\
itusgateway|\
itusrouter)
```

Reply:
> No itus* please.
> Just
> 
> ```
> itusbridge|\
> itusgateway|\
> itusrouter)
> ```

No problems.. Will be changed :)